monoio/src/driver/op/symlink.rs

```diff
@@ -0,0 +1,43 @@
+use std::{ffi::CString, io, path::Path};
+
+use super::{Op, OpAble};
+use crate::driver::util::cstr;
+
+pub(crate) struct Symlink {
+    pub(crate) _from: CString,
```

Comment:
IMO, there is unnecessary to add `_` to the attributes here. I think it's better to remove `_`.